LICENSE

```diff
@@ -1,8 +1,8 @@
 This is the MIT license: http://www.opensource.org/licenses/mit-license.php
 
-Copyright (c) 2017-2020, National Instruments Corp.
+© 2021 National Instruments Corporation. All rights reserved
```

Comment:
Follow the format prescribed in https://opensource.org/licenses/mit-license.php, with `National Instruments Corporation` as the copyright holder.